### PR TITLE
Copy kubectl command ref docs to Hugo site.

### PR DESCRIPTION
**Pull request by steveperry-53**

### CODE REVIEW

**Comment by steveperry-53** [comment]
Things seem to be in order. We can work on the left nav in a separate PR.
https://deploy-preview-8430--kubernetes-io-master-staging.netlify.com/docs/reference/generated/kubectl/kubectl-commands

**Comment by steveperry-53** [comment]
@zacharysarah, Ready for review.

File: content/en/docs/reference/kubectl/overview.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ approvers:
 title: Overview of kubectl
 ---
 
-`kubectl` is a command line interface for running commands against Kubernetes clusters. This overview covers `kubectl` syntax, describes the command operations, and provides common examples. For details about each command, including all the supported flags and subcommands, see the [kubectl](/docs/user-guide/kubectl/) reference documentation. For installation instructions see [installing kubectl](/docs/tasks/kubectl/install/).
+`kubectl` is a command line interface for running commands against Kubernetes clusters. This overview covers `kubectl` syntax, describes the command operations, and provides common examples. For details about each command, including all the supported flags and subcommands, see the [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) reference documentation. For installation instructions see [installing kubectl](/docs/tasks/kubectl/install/).
```
Comment (ajaeger) [inline content/en/docs/reference/kubectl/overview.md]: This file contains more references to /docs/user-guide/kubectl/ - could you update all of them?

Might be a better solution than what I do in https://github.com/kubernetes/website/pull/8442/files

**Comment by zacharysarah** [comment]
@steveperry-53 Looks like the kubectl API reference still doesn't appear. (This is the challenge I ran into, also.)

For comparison, see: https://v1-10.docs.kubernetes.io/docs/reference/generated/kubectl/kubectl/

**Comment by steveperry-53** [comment]
@zacharysarah, This PR is only about the kubectl commands page: https://deploy-preview-8430--kubernetes-io-master-staging.netlify.com/docs/reference/generated/kubectl/kubectl-commands.
The kubectl binary page (that you mention above) is a separate matter, and I'll address it in a different PR. Thanks for fixing the merge conflict.

**Pull request merged**